src/session.ts

```diff
@@ -237,6 +237,7 @@ export async function showBrowser(url: string, title: string, viewer: string | b
             }
             void commands.executeCommand('setContext', 'r.browser.active', false);
         });
+        panel.iconPath = new UriIcon('globe');
```

Comment:
Why do I have

<img width="731" alt="image" src="https://user-images.githubusercontent.com/4662568/130100227-c885e54f-bc4b-410e-a126-3e4dcdd497ad.png">

all these complaints from the eslint?

Reply:
I do not get these.
Sometimes I get random `any type` warnings after e.g. switching between Git branches, and reloading the window usually resolves these.

Reply:
Yes, thanks!